lib/foreman/renderer/scope/macros/snippet_rendering.rb

```diff
@@ -7,15 +7,6 @@ def snippet_if_exists(name, options = {})
             snippet(name, { silent: true }, variables: options[:variables])
           end
 
-          # provide embedded snippets support as simple erb templates
-          def snippets(file, options = {})
-            if source.find_snippet(file)
-              snippet(file.gsub(/^_/, ""), options)
-            else
-              render :partial => "unattended/snippets/#{file}"
```

Comment:
This branch has not been working since 1.15 when the template folder structure was changed so it can be safely removed.